hiroaki-core/src/main/java/me/jorgecastillo/hiroaki/ServerExtensions.kt

```diff
@@ -23,17 +23,31 @@ private fun okHttpClient(
                 .addInterceptor(HttpLoggingInterceptor().setLevel(loggingLevel))
                 .build()
 
+inline fun <reified T> MockWebServer.retrofitService(
+    converterFactory: Converter.Factory? = null,
+    okHttpClient: OkHttpClient? = null
+): T = if (okHttpClient == null) {
```

Comment:
Had to do this `if/else` since `okHttpClient()` is private and cannot be used inside `inline fun`.